app/controllers/api_controller.rb

```diff
@@ -20,6 +20,7 @@ def respond_to_format(response_data, filename)
         send_data export_organizations_as_docx(response_data), filename: "#{filename}.docx" 
       end
       format.markdown {render text: export_organizations_as_markdown(response_data)}
+      format.any {  raise ActionController::RoutingError.new('Not Found') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.